cpp/test/evaluation.cc

```diff
@@ -15,8 +15,8 @@ TEST(EvaluationTest, TestFiveCards) {
   int progress = 0;
   const int total = 2598960;
 
+  std::vector<int> indexCount(7463,0);
```

Comment:
I believe the header `<vector>` is missing.

In addition, if we could use `std::vector<bool>`, it will save us some memory.

Reply:
good point. i'll look at that